### ISSUE TITLE
Can it be used for Laravel 7?

### ISSUE THREAD

**Issue by sprklinginfo** [tolacika/laravel-cron-bundle#1]
Hi there,
tried to install on my Larave 7 project but failed. Is it possible to update the package to Laravel 7? Thanks!

**Comment by tolacika** [comment]
Thanks for the feedback.

Please give me some time, and I will update the package.

**Comment by cgencer** [comment]
i'd like to use it with 7 also. any news?

**Comment by george-raphael** [comment]
Any news? for support in Laravel 7

**Comment by cgencer** [comment]
nope, i simply added my jobs per hand. have much more stuff to work on arm...
